app/src/test/java/com/example/KnowYouLimit/DrinksDataUnitClass.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.KnowYouLimit
+
+class DrinksDataUnitClass {
+
+}
```

Comment:
live data unit test class creation for when JSON link is inserted.

Reply:
I caution against adding an empty class.  If this never gets used, it's clutter, which is technical debt.  Only add this class when it will be used.